src/wizard/unreal/index.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Unreal Engine
+doc_link: https://docs.sentry.io/platforms/unreal/
+support_level: production
+type: framework
```

Comment:
Please place redirects from the old root pages here, and make sure to add redirects for all the moved pages. Apart from links in Sentry docs, there can be blog posts, internal documentation, and bookmarks pointing to those URLs.

```yaml
redirect_from:
  - /platforms/unrealengine/
  - /platforms/native/ue4/
  - /platforms/native/guides/ue4/
```